tests/kernel/udev_no_symlink.pm

```diff
@@ -49,6 +49,7 @@ sub run {
     assert_script_run('journalctl -u detect-part-label-duplicates.service --no-pager | grep "Warning: a high number of partitions uses"');
     assert_script_run('test $(grep -r "E:ID_PART_ENTRY_NAME=primary" /run/udev/data | wc -l) -eq ' . $num_primary);
     assert_script_run('test $(grep -r "E:ID_PART_ENTRY_NAME=openqapart" /run/udev/data | wc -l) -eq ' . $num_openqapart);
+    script_run("ls -l /run/udev/links/*by-partlabel*{primary,logical}/*");
```

Comment:
Try to use same logic for similar following commandsm try to use variable for `ls -l /run/udev/links/*by-partlabel*{primary,logical}/*` and put it into test command (to dont repeat it).

Reply:
The same command is run above with `assert_script_run()`:
`assert_script_run('test $(ls -l /run/udev/links/*by-partlabel*{primary,logical}/* | wc -l) -eq 0');`
=> if it fails the first one the latter one won't run at all. If really needed for debugging, it should be run above. But is really just this one needed to be debugged? Maybe some custom function, which would first run it to see the result and then checked the number? And use this function for all tests?